supplier_app/new_urls/AP_urls.py

```diff
@@ -22,6 +24,9 @@
     url(r'^create$', CompanyCreatorView.as_view(), name='company-create'),
     url(r'^invite$', company_invite, name='company-invite'),
     url(r'^$', BuyerTaxpayersList.as_view(), name='company-list'),
+    url(r'^taxpayerList$', BuyerTaxpayersList.as_view(), name='buyer-taxpayer-list'),
```

Comment:
To be consistent with `change-ap-permission` you should use hyphens instead of camel case

Reply:
That not suppose to be there, I will delete it, thanks!